### ISSUE TITLE
Invalid EXIF data: buffer should start with "Exif".

### ISSUE THREAD

**Issue by michael-ts** [substack/jpeg-marker-stream#3]
I copied the example code into a new file exif.js, and then ran

    node exif.js < myimage.jpg

The EXIF info is returned, but I also get this nasty exception in the output which I cannot catch because it's apparently in async code:

```
C:\Users\michael\git\node_modules\exif-reader\index.js:5
    throw new Error('Invalid EXIF data: buffer should start with "Exif".');
    ^

Error: Invalid EXIF data: buffer should start with "Exif".
    at module.exports (C:\Users\michael\git\node_modules\exif-reader\index.js:5:11)
    at DestroyableTransform.flushMarker (C:\Users\michael\git\node_modules\jpeg-marker-stream\index.js:183:17)
    at DestroyableTransform.write [as _transform] (C:\Users\michael\git\node_modules\jpeg-marker-stream\index.js:36:31)
    at DestroyableTransform.Transform._read (C:\Users\michael\git\node_modules\readable-stream\lib\_stream_transform.js:184:10)
    at DestroyableTransform.Transform._write (C:\Users\michael\git\node_modules\readable-stream\lib\_stream_transform.js:172:83)
    at doWrite (C:\Users\michael\git\node_modules\readable-stream\lib\_stream_writable.js:418:64)
    at writeOrBuffer (C:\Users\michael\git\node_modules\readable-stream\lib\_stream_writable.js:407:5)
    at DestroyableTransform.Writable.write (C:\Users\michael\git\node_modules\readable-stream\lib\_stream_writable.js:334:11)
    at ReadStream.ondata (_stream_readable.js:639:20)
    at emitOne (events.js:116:13)
```

The example conveniently pipes stderr to `/dev/null` hiding the fact that this error occurs.  However, inside my application I don't have the luxury of doing such a thing (I need stderr for other things and can't redirect it, and it's going to spam my logs with spurious errors).

**Comment by strarsis** [comment]
@substack: +1! I am unable to find a way of catching this exception.

**Comment by substack** [comment]
fixed in 1.0.1
